From 59448fccf2982e6859ea42b66ec4df8f57c73be0 Mon Sep 17 00:00:00 2001
From: John Levon <john.levon@joyent.com>
Date: Thu, 22 Aug 2019 16:37:24 +0000
Subject: [PATCH] MANTA-4489 ops is under-sized in a lab deployment Reviewed
 by: Trent Mick <trentm@gmail.com> Reviewed by: Isaac Davis
 <isaac.davis@joyent.com> Approved by: Trent Mick <trentm@gmail.com>

---
 config/services/ops/service.json.lab | 5 +++++
 1 file changed, 5 insertions(+)
 create mode 100644 config/services/ops/service.json.lab

diff --git a/config/services/ops/service.json.lab b/config/services/ops/service.json.lab
new file mode 100644
index 0000000..2a27ab7
--- /dev/null
+++ b/config/services/ops/service.json.lab
@@ -0,0 +1,5 @@
+{
+	"params": {
+		"ram": 4096
+	}
+}
-- 
2.21.0

